classes/form/CustomerForm.php

```diff
@@ -147,55 +147,75 @@ public function validate()
                 'Shop.Notifications.Error'
             ));
         }
-        $this->validateFieldsLengths();
         $this->validateByModules();
 
         return parent::validate();
     }
 
+    /**
+     * @deprecated since 1.7.8.0
+     */
     protected function validateFieldsLengths()
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated and has no effect since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
         $this->validateFieldLength('email', 255, $this->getEmailMaxLengthViolationMessage());
         $this->validateFieldLength('firstname', 255, $this->getFirstNameMaxLengthViolationMessage());
         $this->validateFieldLength('lastname', 255, $this->getLastNameMaxLengthViolationMessage());
     }
 
     /**
+     * @deprecated since 1.7.8.0
+     *
      * @param $fieldName
      * @param $maximumLength
      * @param $violationMessage
      */
     protected function validateFieldLength($fieldName, $maximumLength, $violationMessage)
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated and has no effect since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
         $emailField = $this->getField($fieldName);
         if (strlen($emailField->getValue()) > $maximumLength) {
             $emailField->addError($violationMessage);
         }
     }
 
     /**
+     * @deprecated since 1.7.8.0
+     *
      * @return mixed
      */
     protected function getEmailMaxLengthViolationMessage()
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated and has no effect since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
```

Comment:
If you keep the translator message, this method still has effect :sweat_smile:

Reply:
NOPE !  `has no effect` because  https://github.com/PrestaShop/PrestaShop/blob/develop/classes/form/CustomerForm.php#L150  has been removed  😊